main/src/library/Iterator.flix

```diff
@@ -383,6 +555,8 @@ namespace Iterator {
 	///
 	/// Consumes all elements in the iterator.
 	///
+	/// The original iterator `iter` should not be reused.
+    ///
```

Comment:
Yes, I think the whole file could use a reformat. Sadly we don't have tool support for that yet :)

Reply:
When I open it locally, there's no alignment issue. Should we maybe include `.editorconfig` file in the project such that minor issues like these will be easier to avoid.
```editorconfig
[**.flix]
indent_style = space
indent_size = 4
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
```
Additionally, this goes well with `.gitattributes` file.

Reply:
Looks like there are tabs in the file :(

Is `.editorconfig` a VSCode-specific file type?

Reply:
It's an editor agnostic file that enforces the specific file properties. I think it's up to the editors to to implement it but it has widespread adoption.
[editorconfig's website](https://editorconfig.org/)